arch/arm64/boot/dts/rockchip/overlays/Makefile

```
@@ -136,6 +136,7 @@ dtb-$(CONFIG_CLK_RK3528) += \
 	rk3528-spi0-cs1-enc28j60.dtbo \
 	rk3528-spi0-cs1-mcp2515.dtbo \
 	rk3528-spi0-cs1-spidev.dtbo \
+	rk3528-spi1-cs-gpio.dtbo \
```

```suggestion
	rk3528-spi1-cs-gpio-spidev.dtbo \
```
还有另外两个overlay
可以考虑弄个dtsi